fe/fe-core/src/main/java/com/starrocks/sql/optimizer/operator/logical/LogicalMysqlScanOperator.java

```diff
@@ -54,4 +54,9 @@ public LogicalMysqlScanOperator.Builder withOperator(LogicalMysqlScanOperator op
             return this;
         }
     }
+
+    @Override
+    public LogicalMysqlScanOperator clone() {
+        return new LogicalMysqlScanOperator.Builder().withOperator(this).build();
+    }
```

Comment:
Why you don't use OperatorBuilderFactory?

Reply:
OK. will fix that.